t/09-job_clone.t

```diff
@@ -93,7 +94,19 @@ subtest 'get job' => sub {
     qr/failed to get job '$job_id'/, 'invalid job id results in error';
 
     $job_id = 99937;
-    lives_ok { clone_job_get_job($job_id, $remote, $remote_url, \%options) } 'got job';
+    combined_like { clone_job_get_job($job_id, $remote, $remote_url, \%options) } qr/^$/ , 'got job';
```

Comment:
does this really work? Would this fail if there *is* output? I would assume that /^$/ maybe also matches a newline after actual output

Reply:
It fails if `verbose=>1` is added to the test, so yes, if we added anything else in the future producing output, like other logs in libs, it would fail.